build.rs

```diff
@@ -13,27 +13,26 @@ fn main() {
 
     cmd.current_dir(Path::new("rocksdb"))
         .arg("EXTRA_CFLAGS=-fPIC")
-        .arg("EXTRA_CXXFLAGS=-fPIC")
+        .arg("EXTRA_CXXFLAGS=-fPIC") // -std=c++11 -stdlib=libstdc++
         .arg(format!("INSTALL_PATH={}", out_dir));
 
     if let Ok(jobs) = num_jobs {
         cmd.arg(format!("-j{}", jobs));
     }
 
-    cmd.arg("install-static");
+    cmd.arg("install");
```

Comment:
I'm afraid you can't dynamically link to a library from a static library. Unless you do it at run-time (rather than load-time), using `dlopen` or something. This is definitely a problem, although I see why you wanted to avoid bringing two different allocators into play.